test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -3512,12 +3513,12 @@ def __init__(self, name=None, attrs={}, defaults={}, pbsconf_file=None,
         # If diag is given then bypass parsing pbs.conf
         if self.has_diag:
```

Comment:
I suggest renaming `diag` with `snap` in this file like `self.has_diag` with `self.has_snap`, `self.diag` with `self.snap` etc...

Reply:
I was being lazy :) I'll rename them, thanks